Testreglar/254/Nett/nett-2.5.4a.json

```diff
@@ -0,0 +1,151 @@
+{
+	"id": "nett-2.5.4a",
+	"type": "web",
+	"kravTilSamsvar": "<ul>\r\n<li><span lang=\"NO-BOK\" style=\"font-size: 11.0pt; line-height: 107%; font-family: 'Arial',sans-serif; mso-fareast-font-family: Calibri; mso-fareast-theme-font: minor-latin; mso-bidi-font-family: 'Times New Roman'; mso-ansi-language: NO-BOK; mso-fareast-language: EN-US; mso-bidi-language: AR-SA;\">Funksjonalitet som kan brukes med å bevege enheten eller gjennom brukerbevegelser</span>, kan også brukes med vanlige brukergrensesnittkomponenter og</li>\r\n<li>Brukeren kan slå av bevegelsesaktivering av innhold via\r\n<ul>\r\n<li>innstillinger i brukeragenter som lar brukeren slå av bevegelsesaktivering på systemnivå eller</li>\r\n<li>en mekanisme for å slå av bevegelsesaktivering, og mekanismen ligger på samme side som innholdet med bevegelsesaktivering</li>\r\n</ul>\r\n</li>\r\n<li>Kravet gjelder med mindre\r\n<ul>\r\n<li>Støttet grensesnitt: Bevegelsen er brukt for å betjene funksjonalitet gjennom et grensesnitt som er tilgjengelighetsstøttende</li>\r\n<li>Avgjørende: Bevegelsen er avgjørende for funksjonaliteten, og det vil gjøre aktiviteten ugyldig.</li>\r\n</ul>\r\n</li>\r\n</ul>",
```

Comment:
Der mangler noget i denne sætning: "Avgjørende: Bevegelsen er avgjørende for funksjonaliteten, og det vil gjøre aktiviteten ugyldig." 
I succeskriteriet på engelsk står der "and doing so..." som jeg tænker henviser tilbage til "responding to the motion can be disabled to prevent accidental actuation".